feat(mission): Quarg protect Efreti ships from capture (#7572)
endless-sky/endless-sky: data/korath/korath missions.txt

	on complete
		payment 1000
		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`

mission "Efret: Quarg Rescue"
	boarding
	repeat
	invisible
	deadline 1
	source
		government "Kor Efret"
	on offer
		"old efret rep" = "reputation: Kor Efret"
		# bring it low enough that capping this will result in war
		"reputation: Kor Efret" <?= 1
		conversation
			branch "unfriendly quarg" # if you're already at war with them
				"reputation: Quarg" < 0

> 			`The Quarg have brought reinforcements, probably in reaction to a distress call by the <ship>. They hail you, with barely contained anger in their voice. "Remove yourself from hence, and we may yet forgive your aggression. The Kor Efret are under our protection! Think not even about capturing yonder vessel."`

Reviewer: ```suggestion
			`The Quarg have brought reinforcements, probably in reaction to a distress call by the <ship>. They hail you, with barely contained anger in their voice. "Remove yourself, and we may yet forgive your aggression. The Kor Efret are under our protection! Think not even about capturing yon vessel."`
```

First is extraneous, so can just be cut to avoid the awkwardness.

Second is the more appropriate according to quick Google search ( https://www.shakespeareswords.com/Public/LanguageCompanion/ThemesAndTopics.aspx?TopicId=45 ).

Author: I don't know what's up in the air about anything?

I really like the "from hence," and it sounds a *lot* less archaic than "yonder" or "yon," to me.  But as for the latter yes if I'm doing it wrong I should definitely not be.

			choice
				`  "I had not realized they were under your protection. I will leave at once."`
				`  (Ignore the warning.)`
					goto ignore
				`  "I think you should leave. This is my prize and I'll take what I please."`
					goto refuse
			`  "Leave then anon, ere we change our mind."`
				accept
			label ignore
			action
				"reputation: Quarg" <?= -1
			`  "So be it. You shall pay for your transgression against the Efret!" The Quarg begin powering up their weapons...`
				accept
			label refuse
			action
				"reputation: Quarg" <?= -1

> 			`  "What impetuous evil is this! Nay, we shall not allow you your plunder; such may not be tolerated." The Quarg open fire before you have a chance to disengage.`

Reviewer: ```suggestion
			`  "What impetuous evil is this? Nay, we shall not allow you your plunder; we shall not abide it." The Quarg open fire before you have a chance to disengage.`
```

First is a question, even as presented. Second I think sits better with the rest than "tolerate," and the switch in subject is a stronger declaration of intent that better matches the ensuing destruction. Alternatively, the start could be simply "What impetuous evil!" as a remark of astonishment instead of a question.

Author: I think a question mark would make it sound non-rhetorical.  Questions don't always need question marks; many examples of dated work, not even the Elizabethan I was shooting for here but more, Victorian, say, will end even non-rhetorical questions with periods.  Nor do I agree that the exclamation is undermined by the rhetorical appendage of "is this."

Your second suggestion in this paragraph makes it sound a bit repetitive to me...

				accept
	npc evade
		government "Quarg"
		to spawn
			"reputation: Quarg" >= 0
		personality disables staying entering marked
		fleet
			names "quarg"
			variant
				"Quarg Wardragon" 2
	npc evade
		government "Quarg"
		to spawn
			"reputation: Quarg" < 0
		# they don't just disable.
		personality staying entering heroic marked nemesis
		fleet
			names "quarg"
			variant
				"Quarg Wardragon" 3
	# next time we enter a system
	on enter
		conversation
			branch "already was hostile"
				"reputation: Quarg" < 0
			branch "did not listen to warning"
				"reputation: Kor Efret" < "old efret rep"
			`The Quarg and they seem to be willing to forgive this transgression, this time.`
				decline
			label "did not listen to warning"
			action
				"reputation: Quarg" <?= -1
			`You disregarded the warning of the Quarg... They will probably never forgive you.`
				decline
			label "already was hostile"
			`Is seems like the Quarg will not pursue you... But you may want to refrain from assaulting the Efret in the future.`
			action
				"reputation: Quarg" <?= -1

> 	destination "Kuwaru Efreti"

Reviewer 2: ```suggestion
	destination "Kuwaru Efreti"
	to complete
		never
```
like this

	on complete
		payment 1000
		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`

mission "Efret: Quarg Rescue"
	boarding
	repeat
	invisible
	deadline 1
	source
		government "Kor Efret"
	on offer

> 		"old quarg rep" = "reputation: Quarg"

Reviewer 2: ```suggestion
		"old efret rep" = "reputation: Kor Efret"
```

		# bring it low enough that capping this will result in war

> 		"reputation: Quarg" <?= 1

Reviewer 3: The Quarg's attitude towards the Efreti is 0.01. This means they won't actually care if I accept their warning and then capture the ship anyway.

Reviewer 2: Oh well thats easy to fix but I didnt think it would be 0.01

> 		"reputation: Quarg" <?= 1

Reviewer 2: ```suggestion
		"reputation: Kor Efret" <?= 1
```

		conversation
			branch "unfriendly quarg" # if you're already at war with them
				"reputation: Quarg" < 0

> 			`The Quarg have brought reinforcements, probably in reaction to a distress call by the <ship>. They hail you, with barely contained anger in their voice. "Remove yourself from hence, and we may yet forgive your aggression. The Kor Efret are under our protection! Think not even about capturing yonder vessel."`

Reviewer 3: I assume this starts this way to make it shocking or intimidating? There's usually some small preamble setting the scene of a boarding mission.
I'm not sure this achieves what I'm assuming was the intent. It just seems kind of weird and disjointed.

Reviewer 2: Yeah I probably did not take enough time to write some parts properly
Should prob add you were about to board it or smth

			choice

> 				`  "I had not realized they were under your protection. I will leave at once."`

Reviewer 3: You've got spaces at the beginning here instead of tabs. It looks like this continues throughout.

				`  (Ignore the warning.)`
					goto ignore
				`  "I think you should leave. This is my prize and I'll take what I please."`
					goto refuse

> 			`  "Leaveth then anon, ere we changeth our mind."`

Reviewer 2: Maybe this dialog should be changed a bit since they shoot you on the way out?

Author: I did, down on line 292?  Did I not label it right?

Author: Should probably have the quarg ships that appear not be entering though for this one....

Reviewer 2: Well they say to leave before they change their minds, but they are shooting you all the same

Author: Shouldn't that line be skipped by the `goto refuse` ?

				accept
			label ignore
			action
				"reputation: Quarg" <?= -1

> 			`  "So be it. You shall pay for your transgression against the Efret!" The Quarg begin powering up their weapons...`

Reviewer 3: I didn't say or do anything. For all they know, I'm still making up my mind. No mention of them waiting any amount of time or repeating their message.

Reviewer 2: You dont get to make up your mind, they have had enough patience with you already. But that should be mentioned

Reviewer 3: Yes, if they're not going to wait for me, it should say that it does not take long for them to realise my decision.

				accept
			label refuse
			action
				"reputation: Quarg" <?= -1

> 			`  "What impetuous evil is this! Nay, we shall not allow you your plunder; such may not be tolerated." The Quarg open fire before you have a chance to disengage.`

Reviewer 3: Did the Quarg destroy the ship? Interesting choice. So, is the idea that they'll just keep destroying any Efreti ship I try to board just to deny me the ship and its outfits? How far are they willing to go with this? Are they going to kill all the Efreti? Or, just destroy all their ships?
Also, why don't they do this in the `ignore` route?

Reviewer 2: Because here you're provocking them but I didnt write this one initially I dont think there should be a 100% destruction it should be a 10% chance at most

Reviewer 3: I'm provoking them into killing all these Efreti? Aren't the Quarg supposed to be disciplined or something?

Reviewer 2: True, maybe they just come with one more ship or stare at you angrily

Author: Assuming the player is not bluffing, it's either the Quarg blow up the ship, potentially allowing some Efreti to escape in lifepods, or you explicitly kill them one by one at this point, and they don't want you to be better equipped to kill more in the future.

Reviewer 2: I think the best thing to do is to say "you noticed all Efret mysteriously disapeared before the ship exploded"?

				launch
			label "unfriendly quarg"
			`Three hostile Wardragons arrive in the system, in response to your act of aggression. One hails you. "Your evil deeds against the Efret shall not be tolerated any longer!"`
				accept
	npc evade
		government "Quarg"
		to spawn
			"reputation: Quarg" >= 0
		personality disables staying entering marked
		fleet
			names "quarg"
			variant
				"Quarg Wardragon" 2
	npc evade
		government "Quarg"
		to spawn
			"reputation: Quarg" < 0
		# they don't just disable.
		personality staying entering heroic marked nemesis
		fleet
			names "quarg"
			variant
				"Quarg Wardragon" 3
	# next time we enter a system
	on enter
		conversation
			branch "already was hostile"
				"old quarg rep" < 0
			branch "did not listen to warning"
				"reputation: Quarg" < 0
				"old quarg rep" >= 0
			action
				"reputation: Quarg" = "old quarg rep"

> 			"The Quarg and they seem to be willing to forgive this transgression, this time."

Reviewer 3: This seems incomplete.
Also, use \` around the line, not ".

> 			branch "already was hostile"
> 				"old quarg rep" < 0
> 			branch "did not listen to warning"
> 				"reputation: Quarg" < 0
> 				"old quarg rep" >= 0
> 			action
> 				"reputation: Quarg" = "old quarg rep"
> 			"The Quarg and they seem to be willing to forgive this transgression, this time."
> 				decline
> 			label "did not listen to warning"
> 			action
> 				"reputation: Kor Efret" <?= -1
> 			"You disregarded the warning of the Quarg... They will probably never forgive you."
> 				decline
> 			label "already was hostile"
> 			"Is seems like the Quarg will not pursue you... But you may want to refrain from assaulting the Efret in the future."
> 			action
> 				"reputation: Kor Efret" <?= -1

Reviewer 2: ```suggestion
			branch "already was hostile"
				"reputation: Quarg" < 0
			branch "did not listen to warning"
				"reputation: Kor Efret" < "old efret rep"
			`The Quarg and they seem to be willing to forgive this transgression, this time.`
				decline
			label "did not listen to warning"
			action
				"reputation: Quarg" <?= -1
			`You disregarded the warning of the Quarg... They will probably never forgive you.`
				decline
			label "already was hostile"
			`Is seems like the Quarg will not pursue you... But you may want to refrain from assaulting the Efret in the future.`
			action
				"reputation: Quarg" <?= -1
```

> 	destination "Kuwaru Efreti"

Reviewer 3: This should be near the top of the mission definition, not hte last thing in it.

Reviewer 2: since it doesnt matter its usually at the end in boarding missions I think?

Reviewer 3: Those ones are always declined, but it is possible to accept this one, and it is possible to land on the destination and complete the mission.

Reviewer 2: I had set it to Earth bruh

Author: Was it supposed to be set to Earth?  The comment said that the destination didn't matter

Author: Ohh, I see.  Your edit didn't post before I responded.  Yeah, either one works, though the latter seems better to me.

Reviewer 2: Yeah just use to complete never, the destination doesnt matter only if it isnt in Efret space
